src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/services/ccd/CcdCaseUpdater.java

```diff
@@ -192,11 +218,19 @@ private void updateCaseInCcd(
             );
         } catch (FeignException exception) {
             String msg = format("Service response: %s", exception.contentUTF8());
+            String caseDataContentStr;
+            try {
+                caseDataContentStr = new ObjectMapper().writeValueAsString(caseDataContent);
+            } catch (JsonProcessingException e) {
+                caseDataContentStr = "Exception";
+            }
             log.error(
-                "Failed to update case for {} jurisdiction with case Id {} based on exception record with Id {}. {}",
+                "Failed to update case for {} jurisdiction with case Id {} based on exception record with Id {}. "
+                    + "caseDataContent: {}, {}",
                 exceptionRecord.poBoxJurisdiction,
                 existingCase.getId(),
                 exceptionRecord.id,
+                caseDataContentStr,
```

Comment:
can we log the whole case data?

Reply:
This is actually not case data content, as this is an error response, but I'm not sure we should log it, anyway. Otherwise, we might end up having some sensitive information accidentally logged, as a result of service team's mistake. Thoughts?

Reply:
I guess this served troubleshooting purpose.